src/components/AddTaskForm.tsx

```diff
@@ -192,6 +291,48 @@ export function AddTaskForm() {
                         </MenuList>
                     </Menu>   
             </HStack>
+
+            <HStack justifyContent="space-between" width={"100%"}>
+                    <FormLabel> Assign To: </FormLabel> 
+                    <Menu>           
+                        <MenuButton as={Button} rightIcon={<ChevronDownIcon />}>
+                        <Text id="assignStr"></Text>
+                        </MenuButton>
+                        <MenuList>
+                        {projectUsers.map((user) => 
+                          <MenuItem onClick={() => {
+                            formik.setFieldValue("assignTo", user.username);
+                            updateAssignButton(user.username)
+                            }}
```

Comment:
There should be an "Unassigned" option so users can change the field back to Unassigned